libyul/CMakeLists.txt

```diff
@@ -179,4 +179,23 @@ add_library(yul
 	optimiser/VarNameCleaner.h
 )
 
+set(POLYFILLS arithmetic bitwise comparison conversion interface keccak logical memory)
+message(STATUS "composing ewasm polyfill...")
+foreach(polyfill IN LISTS POLYFILLS)
+	message(STATUS "  reading libyul/backends/wasm/polyfill/${polyfill}.yul -> generating include/generated/ewasm_polyfill_${polyfill}.h")
```

Comment:
This is a bit too verbose for my taste...

Reply:
Also why the `generated` subdirectory?

Reply:
Maybe better to call this `ewasmPolyfills/<name>.h`?

Reply:
And maybe use capitalize the first letter of name (already in the source file).

Reply:
oki